users-service/service.yml

```diff
@@ -26,4 +26,6 @@ messages:
 
 
 dependencies:
-  - 'mongo'
+  mongo:
+    prod: 'ds143588.mlab.com:43588'
+    test: 'localhost:27017'
```

Comment:
Does this setting use the MongoDB running in Docker or the one running on the host machine?